hamsa_lomi/ios/Runner/GoogleService-Info (1).plist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is the file correct here, I mean the (1) part?

Reply:
yes i fix but i didn't push i guess